### ISSUE TITLE
Analysis view should show analysis for selected instruction, not current one

### ISSUE THREAD

**Issue by 10110111** [eteran/edb-debugger#216]
OllyDbg has this useful feature — giving analysis for more than just current instruction. Its selection moves to current instruction on debuggee stop, and you can point on any other and see useful comments about its operands.
EDB needs this too.

**Comment by eteran** [comment]
100% agree, though that will require some basic type of instruction simulation so it can "look ahead" and "look behind".

**Comment by 10110111** [comment]
Not necessarily, it can at least do as in OllyDbg, presenting analysis with current values of registers and adding a red `(current registers)` note. See the screenshot:

![screenshot - 09092015 - 08 17 09 am](https://cloud.githubusercontent.com/assets/6376882/9753883/494175f6-56cb-11e5-950d-f5a12c9595d3.png)
